### ISSUE TITLE
Bug: For "Host", pre-entered text and help seem wrong and misleading

### ISSUE THREAD

**Issue by glorious1** [qdm12/ddns-updater#681]
**TLDR**: The app install interface and tutorial say "@" should precede the hostname.  That doesn't work in my case.

1. Is this urgent: Yes
2. DNS provider(s) you use: Duck DNS
3. Program version: 2.6.0

    

    `Running version 2.6.0 built on 2024-02-03T18:53:40.722Z (commit c338c28)`

4. What are you using to run the container: TrueNAS SCALE UI
5. Extra information (optional)
I struggled setting this up for several days, but it was always "Deploying".  It never made sense to me that the Hostname should be preceded with "@".  The App seemed to be broken because when I tried to check logs, there was no container listed so I couldn't access logs.  Eventually I deleted the app and started over without any "@" in Host field.  Voila.

See also the Tutorial, the instructions for entering Host:
https://www.truenas.com/docs/scale/scaletutorials/apps/communityapps/ddns-updater/#ddns-updater-configuration
 
Logs:

```log

```

Configuration file (**remove your credentials!**):

```json

```

Host OS:

**Comment by qdm12** [comment]
It depends on the provider, duckdns "official" (as in, written for the container, not for the truenas setup which I'm not the author of) is at https://github.com/qdm12/ddns-updater/blob/aa12ccc0b9ed6b929cf4dab22e78bb07ed30cb37/docs/duckdns.md#L24

So technically, it should have a `"host"` set and no `"domain"` set (since it's always duckdns.org). I'm not too sure how they map the setup in truenas to the json configuration file for this container.

Would you please suggest a change at https://github.com/truenas/documentation/edit/master/content/SCALE/SCALETutorials/Apps/CommunityApps/ddns-updater.md and you can refer this issue for the maintainers of the truenas documentation? Especially since I don't use Truenas 😉 Thanks! 👍 

>  The App seemed to be broken because when I tried to check logs, there was no container listed so I couldn't access logs.

`@` is not allowed for the host for duckdns, so the container prints an error and exits, that's why the container is no longer there, and I guess truenas keeps it in a "Deploying" state, since it keeps on exiting as soon as it starts 🤔
